### DISCUSSION TITLE
run prettier on the repository?

### DISCUSSION THREAD

**Discussion by dorianmariecom** [rubygems/rubygems.org#2861] [Ideas]
would make the code consistent with other codebases and well "prettier"

could also be in a before-commit hook and integrated into the CI

what do you think?

**Comment by dorianmariecom** [comment]
I see there is in the `CONTRIBUTING.md`:

> We use RuboCop to enforce a consistent coding style throughout the project.
> Please ensure any changes you make conform to our style standards or else the build will fail.

So that may apply to CSS, JS, and ruby formatting?

**Reply by sonalkr132** [reply]
most of the development is done on ruby side only. css and js are modified/added once every few months, that too is very small chunks. it may be hard to justify the burden of managing and learning one more tool when we may not use it is as much. we also try to keep our build as lean as possible.
